riak/transports/pbc/connection.py

```diff
@@ -165,7 +165,7 @@ def _ssl_handshake(self):
 
                     return True
                 except ssl.SSLError as e:
-                    raise SecurityError(e.library + ": " + e.reason)
```

Comment:
```sh
$ python2 --version
Python 2.7.10
```

I got some sort of exception saying that `library` was not an attribute of `SSLError`. So, I just changed it to pass the exception itself to `SecurityError` and I got a lot more helpful information in the output. I will reproduce and will show the difference in just a sec.